src/explore-education-statistics-admin/src/pages/release/content/contexts/useReleaseContentActions.ts

```diff
@@ -234,6 +239,86 @@ export default function useReleaseContentActions() {
     [dispatch, updateAvailableDataBlocks],
   );
 
+  const addEmbedSectionBlock = useCallback(
+    async ({
+      releaseId,
+      sectionId,
+      sectionKey,
+      request,
+    }: {
+      releaseId: string;
+      sectionId: string;
+      sectionKey: ContentSectionKeys;
+      request: EmbedBlockCreateRequest;
+    }) => {
+      const newBlock = await releaseContentService.addEmbedSectionBlock(
+        releaseId,
+        request,
+      );
+
+      dispatch({
+        type: 'ADD_SECTION_BLOCK',
+        payload: { meta: { sectionId, sectionKey }, block: newBlock },
+      });
+
+      await updateAvailableDataBlocks({ releaseId }); // need this?
+    },
+    [dispatch, updateAvailableDataBlocks],
+  );
+
+  const deleteEmbedSectionBlock = useCallback(
+    async ({
+      releaseId,
+      sectionId,
+      blockId,
+      sectionKey,
+    }: {
+      releaseId: string;
+      sectionId: string;
+      blockId: string;
+      sectionKey: ContentSectionKeys;
+    }) => {
+      await releaseContentService.deleteEmbedSectionBlock(releaseId, blockId);
+      dispatch({
+        type: 'REMOVE_SECTION_BLOCK',
+        payload: { meta: { sectionId, blockId, sectionKey } },
+      });
+
+      await updateAvailableDataBlocks({ releaseId });
```

Comment:
This doesn't need to be called as embed blocks don't affect data blocks. We should remove it from the dependency array of this `useCallback` as well.